load.php

```diff
@@ -137,6 +137,34 @@ function( $module_settings ) {
 	return $modules;
 }
 
+/**
+ * Gets the active and valid performance modules.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $module Slug of the module.
+ * @return bool True if the module is active and valid, otherwise false.
+ */
+function perflab_is_valid_module( $module ) {
+
+	if ( empty( $module ) ) {
+		return false;
+	}
+
+	// Do not load module if no longer exists.
+	$module_file = plugin_dir_path( __FILE__ ) . 'modules/' . $module . '/load.php';
+	if ( ! file_exists( $module_file ) ) {
+		return false;
+	}
+
+	// Do not load module if it cannot be loaded, e.g. if it was already merged and is available in WordPress core.
+	if ( ! perflab_can_load_module( $module ) ) {
+		return false;
+	}
+
+	return true;
```

Comment:
This could be shortened to the following:
```suggestion
	return perflab_can_load_module( $module );
```

Not a big deal though.